src/ontology/fbcv-edit.obo

```diff
@@ -8908,6 +8908,88 @@ is_a: FBcv:0000014 ! structural qualifier
 created_by: http://orcid.org/0000-0002-1373-1705
 creation_date: 2020-12-14T14:39:35Z
 
+[Term]
+id: FBcv:0009000
+name: single-cell RNA-Seq
+namespace: assay_type
+def: "A method for quantifying the transcriptome of individual cells, in which transcripts isolated from single cells are subjected to high-throughput sequencing and mapping to a reference genome." [PMID:28821273]
+xref: OBI:0002631
+is_a: FBcv:0003050 ! assay type
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-07-26T12:18:07Z
+
+[Term]
+id: FBcv:0009001
+name: single-nucleus RNA-Seq
+namespace: assay_type
+def: "A method for quantifying the transcriptome of individual cells, in which transcripts isolated from single nuclei are subjected to high-throughput sequencing and mapping to a reference genome." [PMID:28821273]
+is_a: FBcv:0003050 ! assay type
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-07-26T12:19:45Z
+
+[Term]
+id: FBcv:0009002
+name: cell clustering analysis
+namespace: result_type
+def: "A result that groups individual cells into populations sharing a similar transcriptional profile." [PMID:28821273]
+is_a: FBcv:0003082 ! result type
```

Comment:
I think this is an 'analysis'.
some other stuff under 'result type', e.g. 'differential gene expression analysis' and 'expression clustering' should probably also be under 'analysis' too.